pkg/cloudprovider/providers/gce/gce.go

```diff
@@ -241,6 +241,17 @@ func generateCloudConfig(configFile *ConfigFile) (cloudConfig *CloudConfig, err
 	cloudConfig.UseMetadataServer = true
 
 	if configFile != nil {
+		alphaFeatureGate, err := NewAlphaFeatureGate(configFile.Global.AlphaFeatures)
+		if err != nil {
+			glog.Errorf("Encountered error for creating alpha feature gate: %v", err)
+		}
+		cloudConfig.AlphaFeatureGate = alphaFeatureGate
+	} else {
+		featureMap := make(map[string]bool)
```

Comment:
Actually I mean, 

```Go
featureMap := make(map[string]bool)
cloudConfig.AlphaFeatureGate = &AlphaFeatureGate{featureMap}
if configFile != nil {
   ...
   alphaFeatureGate, err := NewAlphaFeatureGate(configFile.Global.AlphaFeatures)
   if err != nil {
       ...
   }
   cloudConfig.AlphaFeatureGate = alphaFeatureGate
   ...
}
```

Reply:
We don't need two blocks `if configFile != nil`.

Please also squash your commits. Thanks for your contribution.